airflow/providers/amazon/aws/sensors/cloud_formation.py

```diff
@@ -86,8 +95,10 @@ def poke(self, context):
             return False
         raise ValueError(f'Stack {self.stack_name} in bad state: {stack_status}')
 
-    def get_hook(self):
+    def get_hook(self) -> AWSCloudFormationHook:
         """Create and return an AWSCloudFormationHook"""
-        if not self.hook:
-            self.hook = AWSCloudFormationHook(aws_conn_id=self.aws_conn_id, region_name=self.region_name)
+        if self.hook:
```

Comment:
This `if` is because of `None` default value of `hook` and hence mypy complaining for it?

Reply:
Yes. get_hook() returns an actual value, but in the original form mypy hence type error.